### ISSUE TITLE
Word selection fails in Unix (LF) line ending mode

### ISSUE THREAD

**Issue by Albrecht-S** [notepad-plus-plus/notepad-plus-plus#3044]
### Description of the Issue
Word selection with double-click in Unix (LF) line ending mode selects more than a single word at the end of a line. The same works well if line ending mode is set to Windows (CR+LF) or Mac (CR).

This is a regression in notepad++ 7.3.3 (or earlier). I found this after upgrade to 7.3.3, but I can't tell exactly which version I used before (I may have missed one or more version updates).

### Steps to Reproduce the Issue

1. set line ending mode to UNIX (LF) [ German: Bearbeiten/Format Zeilenende/... UNIX (LF) ]
2. double-click last word in line
3. watch selection or press <DEL>

### Expected Behavior
only the double-clicked word should be selected

### Actual Behavior
selects the word + the following (one or more) line breaks (LF) and the next word on a new line. Stops at space or tab though.

### Debug Information
Notepad++ v7.3.3   (32-bit)
Build time : Mar  8 2017 - 03:37:37
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)

### Example text
Copy the following text into notepad++ and double-click the word 'select' in its four occurrences at the line ends:
(1) word select
fails

(2) word select

fails even with empty line

(3) word select
 fails with indentation (selects newline)

(4) word select 
works with a space after 'select' in previous line

Don't forget to set line ending mode to UNIX (LF).

PS: I didn't find a similar issue with "is:issue is:open +word +select". Sorry if I missed one.

**Comment by Albrecht-S** [comment]
Note: example text (4) lost the space after 'select' when posting this issue, so please add one yourself if you like to test.

**Comment by Albrecht-S** [comment]
I verified that this issue is fixed in v7.6 (64-bit)
```
Notepad++ v7.6   (64-bit)
Build time : Nov 13 2018 - 00:12:05
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll 
```

Thanks for fixing this.
